src/Navbar.js

```diff
@@ -0,0 +1,32 @@
+import React from "react"
+import PropTypes from "prop-types"
+import { Link, useMatch, useResolvedPath } from "react-router-dom"
```

Comment:
Also, this one is our fault, if you don't mind can you put these inside of a components directory right under src. So it would be something like src/components/Navbar.js. Same thing with Team.js

Reply:
Sure thing, I'll get right on both of these tasks!